CorsixTH/Lua/utility.lua

```diff
@@ -312,3 +312,68 @@ function hasBit(value, bit)
   local p = 2 ^ bit
   return value % (p + p) >= p
 end
+
+-- Function to convert table to string. Pretty much just used in "config_finder.lua".
+function Table2String(table, recursive)
```

Comment:
Can't you modify print_table to suit your uses instead of writing a whole new implementation?

Reply:
I can try but Table2String returns a string while print_table prints directly to the terminal.
Table2String also outputs a table like this: `{"shift","q"}`
While print_table outputs like this:
`1 shift`
`2 q`
 It seems that having to change print_table would change it's original purpose a little too much.

Would making Table2String a local function inside config_finder.lua work for you? That's the only place it's used right now. I'll also change it to "table_toString" to fit the naming method, as I should of before.